pkg/distributor/distributor.go

```diff
@@ -167,6 +170,32 @@ type Config struct {
 	OTLPConfig OTLPConfig `yaml:"otlp"`
 }
 
+type PushHandlerMetrics struct {
+	pushRequestSizeBytes *prometheus.HistogramVec
+}
+
+func NewPushHandlerMetrics(reg prometheus.Registerer) *PushHandlerMetrics {
+	return &PushHandlerMetrics{
+		pushRequestSizeBytes: promauto.With(reg).NewHistogramVec(prometheus.HistogramOpts{
+			Name:                            "cortex_distributor_push_requests_uncompressed_bytes",
```

Comment:
The naming standard should mention `size` so it is `cortex_distributor_push_requests_uncompressed_size_bytes`

Reply:
Thanks, I have fixed it.